apps/website/src/composables/wallet.ts

```diff
@@ -72,7 +106,7 @@ function getAvailableProviders(ethereum: any) {
   }
 }
 
-async function requestAccount(provider: WalletProvider) {
+async function requestAccount(provider: EthersProvider) {
```

Comment:
As we add others, let's move web ethereum logic to a composable like `ethers.ts`, use `requestEthersAccount`, etc.